src/libponyrt/actor/actor.c

```diff
@@ -15,6 +15,9 @@
 #include <valgrind/helgrind.h>
 #endif
 
+// default actor batch size
+#define PONY_ACTOR_DEFAULT_BATCH 100
```

Comment:
Why the rename from PONY_SCHED_BATCH ?

ACTOR_DEFAULT_BATCH is less meaningful to me than SCHED_BATCH.

Reply:
I had renamed it because it moved from `scheduler.c` to `actor.c`. I'm more than happy to revert it or name it something else.

Reply:
I think keeping PONY_SCHED_BATCH is more clear what it is a batch for...